pyiron_atomistics/calphy/calphy.py

```diff
@@ -0,0 +1,34 @@
+# coding: utf-8
+# Copyright (c) Max-Planck-Institut für Eisenforschung GmbH - Computational Materials Design (CM) Department
+# Distributed under the terms of "New BSD License", see the LICENSE file.
+
+from pyiron_atomistics.calphy.base import CalphyBase
+
+__author__ = "Sarath Menon"
+__copyright__ = (
+    "Copyright 2022, Max-Planck-Institut für Eisenforschung GmbH - "
+    "- Computational Materials Design (CM) Department"
+)
+__version__ = "1.0"
+__maintainer__ = "Sarath Menon"
+__email__ = "s.menon@mpie.de"
+__status__ = "production"
+__date__ = "Feb 14, 2022"
+
+
+class Calphy(CalphyBase):

```

Comment:
I am not sure I follow fully the changes. In principle, I could remove `calphy.py` altogether. Would this be the best idea?

Reply:
Yes, that's want I recommend.  Remove `Calphy` and rename `CalphyBase` to `Calphy`.  I'd probably then also rename the file from `base.py` to `job.py` or something similar, because words like `base`, `generic`, etc. have very little information content.

Reply:
I renamed the files and removed base